tests/utils/managed_deployment.py

```
@@ -0,0 +1,768 @@
+
+        try:
+            url = f"http://localhost:{pf.local_port}/metrics"
+
+            response = requests.get(url, timeout=30)
+            content = None
+            try:
+                content = response.text
+            except ValueError:
+                pass
+
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)